server/routes/controllers/page-loader.ts

```diff
@@ -77,11 +77,17 @@ export class PageLoader {
    *
    * @todo We don't actually need to delete the cache every time though, only on changes to the JSON files.
    */
-  static getPageDataSource(pageRef?: string): object | any[] | null {
-    if (fs.existsSync(`server/data${pageRef}.json`)) {
-      const url = path.join(__dirname, `../../server/data${pageRef}.json`);
-      delete require.cache[url];
-      return require(`../../data${pageRef}.json`);
+  static getPageDataSource(pageRef: string): Promise<unknown> | null {
+    const url = `server/data${pageRef}`;
+    if (fs.existsSync(`${url}.json`)) {
+      const result = new Promise((resolve) => {
+        console.log(`getting ${url}`);
```

Comment:
Could you change this into a `logger.info(more descriptive log)`?

Reply:
that should not have made it into the PR oops 💀